challenge.js

```diff
@@ -42,7 +47,9 @@
  */
 
 // ANSWER
-
+//
+// div p {}
```

Comment:
In CSS styling, we follow the selector string with curly braces containing the styling rules. Here, all we needed was `div p`. This answer satisfies the question, but here is an explanation and example solution to reinforce it:

Using only tag names and parent/child relationships, here is a solution:

`body > div > p`

`>` is the immediate descendant operator in CSS. So we're looking for a `p` that is the child of a `div`, which is the child of the `body`.

Note that this won't necessarily select *only* E.

1